### ISSUE TITLE
Official support for Let's Encrypt Staging certificates

### ISSUE THREAD

**Issue by micheljung** [hetznercloud/hcloud-cloud-controller-manager#781]
### TL;DR

Staging certificates are not officially supported but should and could easily be supported.

### Expected behavior

Especially when setting up a testing environment where you try to automate things, for instance setting up hcloud via terraform, you might do many mistakes and have to retry several times, or add SANs to your certificate, etc.

For this, it would be very convenient (and even recommended) to use Let's Encrypt Staging certificates. This is already possible by setting `load-balancer.hetzner.cloud/http-managed-certificate-acme-staging: true`, however, a comment says:

> This annotation is exclusively for Hetzner internal testing purposes

I ignored this warning because there was no other way, and it worked for months. Recently it was broken for a day or so ("internal error") but now it's working again.

I see no technical challenge for HCCM because AFAIK all it needs to do is to use a different URL when the label is set.

So what speaks against supporting staging certificates officially? If nothing, then please support it :)

**Comment by lukasmetzner** [comment]
Hey,

as stated in the note, this is not officially supported. I have added your request for official support to our internal feedback tracker :)

Best Regards,
Lukas
